pkg/api/controller_test.go

```diff
@@ -2607,38 +2613,43 @@ func TestTLSMutualAuth(t *testing.T) {
 		So(resp, ShouldNotBeNil)
 		So(resp.StatusCode(), ShouldEqual, http.StatusBadRequest)
 
-		// without client certs and creds, should get conn error
-		_, err = resty.R().Get(secureBaseURL)
-		So(err, ShouldNotBeNil)
+		// without client certs and creds, should path auth and get expected 404 code on non-protected endpoint
```

Comment:
```suggestion
		// without client certs and creds, should pass auth and get expected 404 code on non-protected endpoint
```

Applies to multiple comments in this file.

Reply:
Omg sorry for my english=)